projects/clamav/clamav-scanfile-fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+timeout = 5
```

Comment:
I assume you you know the implications of setting this and clamav is intended to process files fast, so anything slower than 5 seconds is a bug.
But just in case: it means that if a target takes more than 5 seconds to process a test case, libFuzzer will stop fuzzing and report the test case as causing a bug.

Reply:
I didn't realize this.  `timeout = 5` was copy-pasted.  However, I think that _may_ be ok.  5 seconds is quite long for scanning a file.